neutralinojs/neutralinojs
# add webview 'debugger' (think chrome dev tool) for window mode (Linux)

![inspector](https://user-images.githubusercontent.com/24475030/75303385-8b544980-587b-11ea-99d2-49c611457da0.png)
## Description
See #208 
Introduce debugger/web dev tools windows using webkitgtk inspector. Can be turned on
and off in settings.json (default is false for off)

## Changes proposed


- add a dev tool that can be turned off and on at runtime (through settings.json)

## How to test it


- build the core-linux build
- go to settings.json, and change enableinspector property to true
- run neutralino and you will see the debugger popup as a separate window
- change enableinspector to false settings.json if you want to turn it off every time you run neutralino

## Next steps


None.

## Deploy notes


Currently, only separate debugger window supported. The tools in the inspector are mostly feature complete and similar to other major browsers (like firefox) web dev tools
